notification-support/src/main/java/com/expediagroup/streamplatform/streamregistry/core/events/NotificationEventUtils.java

```diff
@@ -151,6 +185,12 @@ private static void validateSchemaKey(Schema schema) {
     checkNotNull(schema.getKey().getDomain(), canNotBeNull("key's domain"));
   }
 
+  private static void validateSchemaKey(SchemaKey schemaKey) {
+    checkNotNull(schemaKey, canNotBeNull("schema key"));
```

Comment:
FWIW, this can be `Objects.requireNonNull` if you want to remove Guava dep

Reply:
That's a good idea, even if guava is transitively available in the module.